lab_6_pipeline/pipeline.py

```diff
@@ -62,11 +108,17 @@ def __init__(
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper | None): Analyzer instance
         """
+        self._corpus = corpus_manager
+        self.analyzer = analyzer
 
     def run(self) -> None:
         """
         Perform basic preprocessing and write processed text to files.
         """
+        for article in self._corpus.get_articles().values():
+            to_cleaned(article)
+            article.set_conllu_info(self.analyzer.analyze(split_by_sentence(article.text)))
```

Comment:
no. we have to pass to analyzer a batch of documents and not batch of sentences. this idea should considerably change the logic of this method